### DISCUSSION TITLE
Bulk import does not allow file upload

### DISCUSSION THREAD

**Discussion by georgringer** [pimcore/pimcore#13199] [Q&A / Support]
Hi everybody!

I am developing my  1st site with Pimcore and it looks great for now.  However I currently want to deploy some changes I did to collections, object bricks, ... and I have seen the "bulk export" which exports all kind of stuff into a json. 

The *bulk import* doesn't trigger any file upload, it just does nothing. How can I import the json? It also can be reproduced at https://demo.pimcore.com/admin

thanks for help!

**Comment by dvesh3** [comment]
Hi @georgringer It works for me on latest version 10.5.6 (see below screenshot). When clicking on bulk import and selecting the exported file, uploads the file and shows the modal to select the definitions to import. Could you please check your console or network for any error related to the server? thanks!
![image](https://user-images.githubusercontent.com/5137917/191197729-d5444577-e682-4b97-b59c-822c9f05c007.png)

**Reply by georgringer** [reply]
thanks for the quick reply. can you reproduce that it doesn't work on latest chrome? I get it working on Firefox
